doc/quick-start/README.md

```diff
@@ -1,52 +1,19 @@
 # Quick Start
 
-## Build an Xline docker image
+## Run Xline from a pre-built image
 
 ```bash
 # Assume that docker engine environment is installed.
 
-# clone source code
-git clone https://github.com/datenlord/Xline
-cd Xline
-
-# build docker image
-# you may need to add sudo before the command to make it work
-docker build . -t datenlord/xline:latest -f doc/quick-start/Dockerfile
-```
-
-## Start Xline servers
-
-``` bash
-cp ./xline-test-utils/{private,public}.pem ./scripts
-
-./scripts/quick_start.sh
-```
-
-## Send Etcd requests
-
-``` bash
-# Set Key A's value to 1
-docker exec node4 /bin/sh -c "/usr/local/bin/etcdctl --endpoints=\"http://172.20.0.3:2379\" put A 1"
-
-# Get Key A's value
-docker exec node4 /bin/sh -c "/usr/local/bin/etcdctl --endpoints=\"http://172.20.0.3:2379\" get A"
-```
-
-## Validation Test
-
-```bash
-docker cp node1:/usr/local/bin/lock_client ./scripts
-
-./scripts/validation_test.sh
-```
-
-## Benchmark
-
-```bash
-./scripts/benchmark.sh
+docker run -it --name=xline datenlord/xline \
+  xline \
+  --name xline \
+  --storage-engine rocksdb \
+  --members xline=127.0.0.1:2379 \
+  --data-dir /usr/local/xline/data-dir
```

Comment:
According to #335, Xline currently does not support single node cluster.
This PR should be merged once the single node cluster implementation is completed.

Reply:
This PR should rebase on #346